.github/workflows/metadata_upload_manifest_dagger.yml

```diff
@@ -0,0 +1,22 @@
+name: Metadata Service CI
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  metadata_service_ci:
+    name: Upload modified metadata files to GCS
+    runs-on: medium-runner
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+      - name: Run metadata upload pipeline
+        id: metadata-upload-pipeline
+        uses: ./.github/actions/run-dagger-pipeline
+        with:
+          subcommand: "metadata upload prod-airbyte-cloud-connector-metadata-service"
```

Comment:
@bnchrch do you confirm this is the destination bucket we want?

Reply:
Lets use this bucket while we are still developing
https://console.cloud.google.com/storage/browser/dev-airbyte-cloud-connector-metadata-service;tab=objects?forceOnBucketsSortingFiltering=false&project=dev-ab-cloud-proj&prefix=&forceOnObjectsSortingFiltering=false